src/main/java/uk/gov/hmcts/reform/professionalapi/domain/LanguagePreference.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmcts.reform.professionalapi.domain;
+
+public enum LanguagePreference {
+    EN

```

Comment:
Should also have CY here too.

Reply:
Please refactor after this when going to preview branch.

Reply:
Rosemary?